Mage/src/main/java/mage/filter/predicate/permanent/ControllerPredicate.java

```diff
@@ -69,6 +69,10 @@ public boolean apply(ObjectPlayer<Controllable> input, Game game) {
                     return true;
                 }
                 break;
+            case ACTIVE:
+                if (object.getControllerId().equals(game.getActivePlayerId())) {
+                    return true;
```

Comment:
I can't believe I forgot the break statement. Though it appears that is not the only issue with it. Now that the break statement is added the ability can no longer be activated. So there is something I'm no seeing or understanding. If I comment out the controller predicate line in nettling imp I can activate and pick a target again.